### PR TITLE
Creating Issue templates

### PR DESCRIPTION
**Pull request by hansinikarunarathne**
**Which issue is resolved by this Pull Request:**
- Created a Issue Template 

**Description of your changes:**
- Created issue templates
- Added contact links from `config.yml`

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/issue-report.yml [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+          required: true
+        - label: Are you posting in the right repository ?
+          required: true
+  - type: dropdown
+    id: version
+    attributes:
+      label: Version
+      description: What version of our software are you running?
+      options:
+        - 1.8
```
Comment (juliusvonkohout) [inline .github/ISSUE_TEMPLATE/issue-report.yml:25]: Lets start with 1.8 and 1.9 only

File: .github/ISSUE_TEMPLATE/issue-report.yml [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+      label: Describe your issue
+      placeholder: When I Procced this issue is occured. 
+    validations:
+      required: true
+  - type: textarea
+    id: steps
+    attributes:
+      label: Steps to reproduce the issue
+      placeholder: |
+        1. Try this ...
```
Comment (juliusvonkohout) [inline .github/ISSUE_TEMPLATE/issue-report.yml:50]: Hello,

1. Is this a Kubeflow issue ?
2. Is this for Kubeflow development ?
3. Are you posting in the right repository ?
4. Did you follow the installation guide https://github.com/kubeflow/manifests?tab=readme-ov-file ? 
5. Is the issue report properly structured and detailed with version numbers ?

If not please close or redo the issue and join our slack channel `wg-manifests` here https://www.kubeflow.org/docs/about/community/. This link also contains our meeting schedule.

File: .github/ISSUE_TEMPLATE/issue-report.yml [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+  - type: dropdown
+    id: assign
+    attributes:
+      label: "Would you like to work on this issue?"
+      options:
+        - "Yes"
+  - type: markdown
+    attributes:
+      value: |
+        > **Note:** Please note that you have to fill required fields to post an Issue.
```
Comment (juliusvonkohout) [inline .github/ISSUE_TEMPLATE/issue-report.yml:67]: Hello,

1. Is this a Kubeflow issue ?
2. Is this for Kubeflow development ?
3. Are you posting in the right repository ?
4. Did you follow the installation guide https://github.com/kubeflow/manifests?tab=readme-ov-file ? 
5. Is the issue report properly structured and detailed with version numbers ?

If not please close or redo the issue and join our slack channel `wg-manifests` here https://www.kubeflow.org/docs/about/community/. This link also contains our meeting schedule.

This should be our checklist.

File: .github/ISSUE_TEMPLATE/issue-report.yml [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+    validations:
+      required: true
+  - type: textarea
+    id: screenshots
+    attributes:
+      label: Put here any screenshots or videos (optional)
+  - type: dropdown
+    id: assign
+    attributes:
+      label: "Would you like to work on this issue?"
```
Comment (juliusvonkohout) [inline .github/ISSUE_TEMPLATE/issue-report.yml:61]: ""Would you like to work on this issue?" This is also a good question

File: .github/ISSUE_TEMPLATE/config.yml [hunk 5]
```diff
@@ -0,0 +1,6 @@
+blank_issues_enabled: false
+contact_links:
+  - name: Join The Community
+    url: https://www.kubeflow.org/docs/about/community/
+    about: Join our slack channel and access our meeting schedule.
```
Comment (juliusvonkohout) [inline .github/ISSUE_TEMPLATE/config.yml:5]: join our slack channel `wg-manifests` here https://www.kubeflow.org/docs/about/community/. This link also contains our meeting schedule.

**Review by juliusvonkohout — CHANGES_REQUESTED** [review]
as stated in the comments

File: .github/ISSUE_TEMPLATE/issue-report.yml (hunk 2 above)
Comment (juliusvonkohout) [reply .github/ISSUE_TEMPLATE/issue-report.yml:50]: Ah sorry, that probably belongs to the required fields checklist below.

File: .github/ISSUE_TEMPLATE/issue-report.yml (hunk 1 above)
Comment (hansinikarunarathne) [reply .github/ISSUE_TEMPLATE/issue-report.yml:35]: Changes the versions

**Review by juliusvonkohout — APPROVED** [review]

**Pull request merged**